### ISSUE TITLE
Warning: Addon doesn't work in NVDA 2023.2 beta1 with One Core and Speak NG. NVDA is silenced completly with this addon installed

### ISSUE THREAD

**Issue by Carlos-EstebanM** [slohmaier/LanguageIdent#2]
After of install this addon from the addon Store in NVDA 2023.2 beta1, NVDA can't say nothing. Tested with the synthesizers Windows One Core and Espeak NG in Windows 11 22 h2.
Here is the NVDA log of the session with the addon install.
Please check this problem before of NVDA 2023.2 stable, more users can install this addon and have a bad experience.
[nvda-old.log](https://github.com/slohmaier/LanguageIdent/files/12209611/nvda-old.log)

**Comment by sevapopov2** [comment]
I have the same problem. I have tried installing and using it and my NVDA is silent. It is really frustrating, @slohmaier could you please fix that? NVDA 2023.2 stable.

**Comment by sevapopov2** [comment]
@slohmaier I have also checked, add-on doesn't work in NVDA 2023.3 RC 2.

**Comment by nvda-ru** [comment]
Hi!
The add-on also does not work on NVDA 2023.3.
After installing the add-on, NVDA starts, but all speech synthesizers stop working. Everything is silent.
